packages/seal/src/client.ts

```diff
@@ -430,4 +432,37 @@ export class SealClient {
 				return derivedKeys;
 		}
 	}
+
+	/**
+	 * Create a new session key with defined Sui Client.
+	 * @param address - The sender adress
+	 * @param packageId - The package ID for Seal approval
+	 * @param ttlMin - The TTL of the session key.
+	 * @param signer - The optional signer to use.
+	 * @returns The session key.
+	 */
+	createSessionKey(
```

Comment:
this only saves passing the suiclient, no? people could still use the direct new SessionKey and import

Reply:
correct